modules/pricer/src/test/java/com/opengamma/strata/pricer/curve/CalibrationDiscountingBrlTest.java

```diff
@@ -0,0 +1,296 @@
+          .nodes(ALL_NODES).build();
+  private static final RatesCurveGroupDefinition CURVE_GROUP_DEFN =
+      RatesCurveGroupDefinition.builder()
+          .name(CURVE_GROUP_NAME)
+          .addCurve(CURVE_DEFN, BRL, BRL_CDI).build();
+  
+  // Constants
+  private static final double TOLERANCE_PV = 1.0E-6;
+  private static final double TOLERANCE_DELTA = 1.0E3;
+  
+  //-------------------------------------------------------------------------
+  public void calibration_present_value() {
+    RatesProvider result2 =
+        CALIBRATOR.calibrate(CURVE_GROUP_DEFN, ALL_QUOTES, REF_DATA);
+    // Test PV
+    CurveNode[] dscNodes = CURVES_NODES.get(0).get(0);
+    List<ResolvedTrade> dscTrades = new ArrayList<>();
+    for (int i = 0; i < dscNodes.length; i++) {
+      dscTrades.add(dscNodes[i].resolvedTrade(1d, ALL_QUOTES, REF_DATA));
+    }
+    // OIS
+    for (int i = 0; i < OIS_NB_NODES; i++) {
+      MultiCurrencyAmount pvIrs = SWAP_PRICER.presentValue(
+          ((ResolvedSwapTrade) dscTrades.get(i)).getProduct(), result2);
+      assertEquals(pvIrs.getAmount(BRL).getAmount(), 0.0, TOLERANCE_PV);
+    }
+  }
+  
+  //-------------------------------------------------------------------------
+  public void calibration_transition_coherence_par_rate() {
+    RatesProvider provider =
+        CALIBRATOR.calibrate(CURVE_GROUP_DEFN, ALL_QUOTES, REF_DATA);
+    ImmutableList<ResolvedTrade> dscTrades = CURVE_GROUP_DEFN.resolvedTrades(ALL_QUOTES, REF_DATA);
+    
+    for (int loopnode = 0; loopnode < dscTrades.size(); loopnode++) {
+      PointSensitivities pts = SWAP_PRICER.parRateSensitivity(
+          ((ResolvedSwapTrade) dscTrades.get(loopnode)).getProduct(), provider).build();
+      CurrencyParameterSensitivities ps = provider.parameterSensitivity(pts);
+      CurrencyParameterSensitivities mqs = MQC.sensitivity(ps, provider);
+      assertEquals(mqs.size(), 1);
+      CurrencyParameterSensitivity mqsDsc = mqs.getSensitivity(ALL_CURVE_NAME, BRL);
+      assertTrue(mqsDsc.getMarketDataName().equals(ALL_CURVE_NAME));
+      assertTrue(mqsDsc.getCurrency().equals(BRL));
+      DoubleArray mqsData = mqsDsc.getSensitivity();
+      assertEquals(mqsData.size(), dscTrades.size());
+      for (int i = 0; i < mqsData.size(); i++) {
+        assertEquals(mqsData.get(i), (i == loopnode) ? 1.0 : 0.0, TOLERANCE_DELTA);
+      }
+    }
+  }
+  
+  //-------------------------------------------------------------------------
+  public void calibration_market_quote_sensitivity() {
+    double shift = 1.0E-6;
+    Function<MarketData, RatesProvider> f =
+        marketData -> CALIBRATOR.calibrate(CURVE_GROUP_DEFN, marketData, REF_DATA);
+    calibration_market_quote_sensitivity_check(f, shift);
+  }
+  
+  //-------------------------------------------------------------------------
+  private void calibration_market_quote_sensitivity_check(
+      Function<MarketData, RatesProvider> calibrator,
+      double shift) {
```

Comment:
When method parameters are split onto multiple lines, there should be a blank line after the `{`